SimpleCom/SimpleCom.cpp

```diff
@@ -177,7 +177,21 @@ int _tmain(int argc, LPCTSTR argv[])
 	SetCommMask(hSerial, EV_RXCHAR);
 	SetupComm(hSerial, 1, 1);
 
+	COMMTIMEOUTS comm_timeouts;
+	GetCommTimeouts(hSerial, &comm_timeouts);
+	comm_timeouts.ReadIntervalTimeout = 1;
```

Comment:
As I mentioned in https://github.com/YaSuenag/SimpleCom/issues/2#issuecomment-813768777 , why did you set `1` to `ReadIntervalTimeout`? I think it can be zero, and it is nature.

Reply:
When other programs set `ReadIntervalTimeout=MAXDWORD`, on exit, and if the next user of the COM port doesn't set it to something else, then "the read operation is to return immediately with the bytes that have already been received, even if no bytes have been received".

"A value of zero indicates that interval time-outs are not used." I don't think this is what is wanted either. With a value of `1` the thread loop will timeout and have a chance to address other flags such as `terminated` at least every 1 ms. If anything, it probably could be increased to 100 ms or so, but at a 1 ms period, the CPU issue was resolved.

Reply:
I guess the behavior (the loop is not so busy even if `ReadIntervalTimeout` is set to 1ms) is due to `ReadTotalTimeoutConstant`. You mentioned `terminate` flag for example, but I don't think so because SimpleCom would issue `CancellIoEx()` for `hSerial` (handle for serial line) after terminating StdInRedirector.
In addition, I'm concern the loop would wakeup excessively with 1ms timeout.

Thus I think it is prefer to disable timeout. What do you think?

Reply:
It's worth trying both 0 and 100 to see if there's any difference.